store/CSDK/main.cpp

```diff
@@ -224,8 +406,81 @@ bool readFromS3(JNIEnv *env, char *argv[]) {
         printf("%s\t", carbonRow.getDecimal(10));
         printf("%f\t", carbonRow.getFloat(11));
         printf("\n");
+        env->DeleteLocalRef(row);
+        env->DeleteLocalRef(array1);
+    }
+    gettimeofday(&read, NULL);
+    time = 1000000 * (read.tv_sec - start.tv_sec) + read.tv_usec - start.tv_usec;
+    printf("total lines is %d: build time: %lf, read time is %lf s, average speed is %lf records/s\n",
+           i, buildTime, time / 1000000.0, i / (time / 1000000.0));
+
+    reader.close();
+}
+
+/**
+ * read data from S3
+ * parameter is ak sk endpoint
+ *
+ * @param env jni env
+ * @param argv argument vector
+ * @return
+ */
+bool readPerformanceFromS3(JNIEnv *env, char *argv[]) {
```

Comment:
all internal comparision performance test cases are not needed.

Reply:
I want to test for big data(more than 100 million rows), I change the test case name, is it ok?